index.html

```diff
@@ -29,20 +35,27 @@
   <p>
     Several of our community coordinators have contact email addresses for players who prefer to connect
     via email.
-    <ul>{% for contact in site.data.contacts %}
-      <li><a href="mailto:{{ contact.email }}">{{ contact.agent }}</a> ({{ contact.area }}) {% if contact.form != nil %} <a href="{{contact.form}}">Sign-up form</a>{% endif %}</li>{% endfor %}
+    <ul>
+        {% assign contacts = site.data.contacts | sort: 'area' %}
+        {% for contact in contacts %}
+        <li>
+            <b>{{ contact.area }}</b>
```

Comment:
Hmm...if you're making this more area-oriented, we could just move the individuals info into the communities file and render just a single list of communities + their coordinators.